src/style.css

```diff
@@ -0,0 +1,34 @@
+body {
+  margin: 0;
+}
+
+.stars--0,
```

Comment:
<img width="1236" alt="Screenshot 2023-04-14 at 16 35 17" src="https://user-images.githubusercontent.com/53228259/232058388-35f5fe25-b2c0-46eb-b8e2-325fd1984bdd.png">
justify-content: flex-start and flex-direction: row do nothing
and delete all this selectors, use class stars with display: flex